.github/CODEOWNERS

```diff
@@ -2152,6 +2149,9 @@ x-pack/test/api_integration/apis/management/index_management/inference_endpoints
 /x-pack/test/api_integration/apis/security_solution @elastic/security-solution
 /x-pack/test/functional/es_archives/auditbeat/default @elastic/security-solution
 /x-pack/test/functional/es_archives/auditbeat/hosts @elastic/security-solution
+/x-pack/test_serverless/functional/test_suites/security/constants.ts @elastic/security-solution
+/x-pack/test_serverless/api_integration/test_suites/security @elastic/security-solution
+/x-pack/test_serverless/**/test_suites/security/platform_security/ @elastic/kibana-security
```

Comment:
I moved this here because, IIUC, this file is parsed sequentially. Our team needs to own the platform_security folder nested under the security solution folder.